dashboard/pages/Home.py

```diff
@@ -1,28 +1,31 @@
-"""Main page of the dashboard."""
+"""Home page of the dashboard."""
 
 
 from dotenv import load_dotenv
 import streamlit as st
-
 import database
 import charts
 
 
-if __name__ == "__main__":
+def show_home():
 
     load_dotenv()
```

Comment:
You may be calling load_dotenv multiple times, which is redundant, have it run once in your main function or when it runs

Reply:
Okay sounds good, I've moved it to `app.py`